zerver/actions/create_user.py

```diff
@@ -359,6 +373,14 @@ def notify_created_user(user_profile: UserProfile) -> None:
         event = dict(type="realm_user", op="add", person=person_for_without_real_email_access_users)
         send_event_on_commit(user_profile.realm, event, user_ids_without_real_email_access)
 
+    if user_ids_without_access_to_created_user:
+        event = dict(
+            type="realm_user",
+            op="add",
+            person=get_data_for_inaccessible_user(user_profile.id, True, user_profile.realm),
```

Comment:
This feels hard to read. We should change the parameter order and use mandatory kwargs, for example: `get_data_for_inaccessible_user(user_profile.realm, user_profile.id, user_is_active=True)`

Reply:
We have removed the `is_active` parameter as we decided to pass it as `True` for all users.